<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicTransformWidget</class>
 <widget class="QWidget" name="MusicTransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>格式转换</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>475</width>
     <height>364</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>格式转换</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputLineEdit">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>65</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>65</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输入路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputLineEdit">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>95</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>95</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输出路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>65</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>95</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="MusicGifLabelWidget" name="loadingLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>420</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>475</width>
     <height>339</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="formatCombo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>175</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>比特率(kbps)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hzCombo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="kbpsCombo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>175</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>频率(Hz)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>175</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>声道设置(M/S)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>转换格式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="msCombo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>175</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="transformButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>立即转换</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="folderBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>添加输入目录(用于批量转换)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="krc2lrcBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>酷狗KRC转化为LRC</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>inputLineEdit</zorder>
  <zorder>label</zorder>
  <zorder>outputLineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>inputButton</zorder>
  <zorder>outputButton</zorder>
  <zorder>listWidget</zorder>
  <zorder>formatCombo</zorder>
  <zorder>label_4</zorder>
  <zorder>hzCombo</zorder>
  <zorder>kbpsCombo</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_3</zorder>
  <zorder>msCombo</zorder>
  <zorder>transformButton</zorder>
  <zorder>folderBox</zorder>
  <zorder>krc2lrcBox</zorder>
  <zorder>loadingLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicGifLabelWidget</class>
   <extends>QLabel</extends>
   <header>musicgiflabelwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
